client/src/components/NavBar/NavBar.module.scss

```
@@ -0,0 +1,46 @@
+@import "../../global.scss";
+
+.NavBar {
+    background-position: center;
+    background-size: cover;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100vw;
```

Ist das nötig? Theoretisch müsste das eh die gesamte Breite einnehmen.

Ansonsten wären 100% ein bisschen eleganter
```suggestion
    width: 100%;
```